tests/contrib/test_gce.py

```diff
@@ -14,102 +14,195 @@
 
 """Unit tests for oauth2client.contrib.gce."""
 
-import json
 from six.moves import http_client
-from six.moves import urllib
 import unittest2
-
+import time
 import mock
+import json
 
 import httplib2
-from oauth2client._helpers import _to_bytes
-from oauth2client.client import AccessTokenRefreshError
-from oauth2client.client import Credentials
-from oauth2client.client import save_to_well_known_file
-from oauth2client.contrib.gce import _DEFAULT_EMAIL_METADATA
-from oauth2client.contrib.gce import _get_service_account_email
-from oauth2client.contrib.gce import _SCOPES_WARNING
-from oauth2client.contrib.gce import AppAssertionCredentials
-
+from oauth2client.client import HttpAccessTokenRefreshError,\
```

Comment:
Trailing backslashes never please. From the [Google Python style guide][1]:

> Do not use backslash line continuation.

----

Also, no comma imports (again from [style guide][2]):

> Imports should be on separate lines.
>
> Yes:
> ```python
> import os
> import sys
> ```
>
> No:
> ```python
> import os, sys
> ```

[1]: https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length
[2]: https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting